scylla-cdc-driver3/src/main/java/com/scylladb/cdc/cql/driver3/Driver3SchemaBuilder.java

```diff
@@ -95,13 +95,14 @@ private void generatePrimaryKeyColumns() {
         ChangeSchema.DataType dataType = translateColumnDataType(driverDefinition.getType());
         ChangeSchema.DataType baseType = null;
         ChangeSchema.ColumnType baseTableColumnType = ChangeSchema.ColumnType.REGULAR;
-        boolean baseIsNonfrozenList = false;
         if (baseTablePartitionKeyColumnNames.contains(columnName)) {
             baseTableColumnType = ChangeSchema.ColumnType.PARTITION_KEY;
         } else if (baseTableClusteringKeyColumnNames.contains(columnName)) {
             baseTableColumnType = ChangeSchema.ColumnType.CLUSTERING_KEY;
         } else {
-            /* Calculate whether corresponding base column is a non-frozen list.
+            /* Fill in the baseType field. One of its uses is to determine
+             * if the base column is a non-frozen list. Below is commentary
+             * why it's safe to use this value:
```

Comment:
"Why it's safe to use this value" - currently the base type for the column might be stale... (from different schema than cdc log type). In my next PRs I will fix it (cdc log schema and base table schema always consistent) and this commentary will not be needed.